packages/webapp/package.json

```diff
@@ -34,7 +34,8 @@
     "react-intersection-observer": "^8.34.0",
     "react-modal": "^3.15.1",
     "react-parallax-tilt": "^1.7.10",
-    "react-query": "3.39.1",
+    "@tanstack/react-query": "^4.36.1",
+    "@tanstack/react-query-devtools": "^4.35.3",
```

Comment:
My only comment is about this, is it intended to add the devtools to production dependencies?

Reply:
That's a good point, it's from their migration script, but indeed could be in dev deps for all I know.
I'll try and refactor :)

Reply:
I think you will have problems with imports in production then, worth a try. But there is no need to put it in dev dependencies since it is not included in production bundle [see here](https://tanstack.com/query/v4/docs/react/devtools#:~:text=By%20default%2C%20React%20Query%20Devtools%20are%20only%20included%20in%20bundles%20when%20process.env.NODE_ENV%20%3D%3D%3D%20%27development%27%2C%20so%20you%20don%27t%20need%20to%20worry%20about%20excluding%20them%20during%20a%20production%20build.)

Reply:
Oh that's even better, let's leave it for now then.

Reply:
That's good to see.

I do miss the possibility to wrap a import statement in a try...catch like I did in python when I wanted to import a dependency only during development